build_tools/benchmarks/common/benchmark_presentation.py

```diff
@@ -457,41 +516,41 @@ def categorize_benchmarks_into_tables(benchmarks: Dict[
     tables.append(_sort_benchmarks_and_get_table(similar, size_cut))
   if raw:
     tables.append(md.header("Raw Latencies", 3))
-    raw_list = [
-        (_make_series_link(k), f"{_get_fixed_point_str(v.mean_time / 1e6)}",
-         f"{_get_fixed_point_str(v.median_time / 1e6)}",
-         f"{_get_fixed_point_str(v.stddev_time / 1e6)}")
-        for k, v in raw.items()
-    ]
+    raw_list = [(_make_series_link(name=v.name, series_id=k),
+                 f"{_get_fixed_point_str(v.mean_time / 1e6)}",
+                 f"{_get_fixed_point_str(v.median_time / 1e6)}",
+                 f"{_get_fixed_point_str(v.stddev_time / 1e6)}")
+                for k, v in raw.items()]
     tables.append(
         _add_header_and_get_markdown_table(BENCHMARK_RESULTS_HEADERS,
                                            raw_list,
                                            size_cut=size_cut))
   return "\n\n".join(tables)
 
 
-def _sort_metrics_objects_and_get_table(metrics_objs: Dict[str, T],
-                                        mapper: MetricsToTableMapper[T],
-                                        headers: Sequence[str],
-                                        size_cut: Optional[int] = None) -> str:
+def _sort_compilation_metrics_objects_and_get_table(
```

Comment:
This method is de-generalized to save some non-trivial works to handle the series name at line 553. We can refactor later if we ever want to reuse this method on non-CompilationMetrics objects.